DOC-2283: replace RediSearch with Search and Query (#3675)
RediSearch/RediSearch: docs/docs/reference/payloads.md

The payload feature is deprecated in 2.0
{{% /alert %}}
    

> Usually, Search and Query stores documents as hash keys. But if you want to access some data for aggregation or scoring functions, we might want to store that data as an inline payload. This will allow us to evaluate properties of a document for scoring purposes at very low cost.

Reviewer: ```suggestion
Usually, Search and Query stores documents as hashes or JSON. But if you want to access some data for aggregation or scoring functions, we can store that data as an inline payload. This will allow us to evaluate the properties of a document for scoring purposes at a very low cost.
```

Author: Added to PR.
